pkg/dependency/parser/nodejs/npm/parse.go

```diff
@@ -345,6 +345,10 @@ func (p *Parser) pkgNameFromPath(pkgPath string) string {
 	// node_modules/function1
 	// node_modules/nested_func/node_modules/debug
 	if index := strings.LastIndex(pkgPath, nodeModulesDir); index != -1 {
+		if index+len(nodeModulesDir) == len(pkgPath) {
+			return ""
+		}
+
```

Comment:
Is it possible to add a warning here since the package-lock.json file is invalid in this case?